irohad/main/iroha_conf_loader.hpp

```diff
@@ -106,6 +108,16 @@ inline rapidjson::Document parse_iroha_config(const std::string &conf_path) {
                    ac::type_error(mbr::MstSupport, kBoolType));
   ac::assert_fatal(doc[mbr::MstExpirationTime].IsUint(),
                    ac::type_error(mbr::MstExpirationTime, kUintType));
+
+  ac::assert_fatal(doc.HasMember(mbr::MaxRoundsDelay),
```

Comment:
This will break the old config support. I suggest using a default when the key is not set in file. But since it is not trivial with current config parser, I would postpone it till the logger trunk is merged, because there it can be done easily.